cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -522,19 +531,23 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
                     </div>
                 </section>
                 <!-- /.next-steps -->
-                <div class="note">
-                    <p class="alert"><strong>Check your credit report.</strong> If you haven’t <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">checked your credit report recently</a>, do so now. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.</p>
-                </div> <!-- /.note -->
+                <div class="block
+                            block__border-top
+                            block__flush-top
+                            block__padded-top
+                            note-ctr">
+                    <div class="note">
+                        <p class="alert"><strong>Check your credit report.</strong></p>
+                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
+                    </div> <!-- /.note -->
+                </div>
                 <section id="about" class="about wrapper">
                     <div class="content-l">
                         <div class="about-data content-l_col content-l_col-2-3">
                             {% include "_templates/ratings-form.html" %}
                         </div>
                         <div class="content-l_col content-l_col-1-3">
-                            <div class="o-form
-                                        block
-                                        block__padded-top
-                                        block__flush-top">
+                            <div class="o-form">
                             <h3 class="subhead">About our data source for this tool</h3>
```

Comment:
Indent contents for the o-form div.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/805de5edd2f3f85ba299ecef445c033fc1e117a9.